overlord/servicestate/service_control.go

```diff
@@ -59,6 +59,7 @@ type ServiceAction struct {
 	// "reload-or-restart" actions, and when set it restarts also enabled
 	// non-running services, otherwise these services are left inactive.
 	RestartEnabledNonActive bool `json:"restart-enabled-non-active,omitempty"`
+	wrappers.ScopeOptions
```

Comment:
I think service action is also used in quota to trigger a restart, I guess the scope there woudl only affect system services?

Reply:
If nothing is provided in ScopeOptions then it will act on everything, to be backwards compatible with the previous behaviour